README.md

```diff
@@ -26,7 +27,7 @@ inefficient code. Finding this code is usually pretty difficult.
 5. Type in function that you would like to start tracing.
 6. Start tracing clicking green button.
 
-Example rebar2 config entry:
+Example rebar3 config entry:
```

Comment:
ah, true
actually I changed this because xprof's own `rebar.config` is in the `rebar3` format so currently we don't support being included in a `rebar2` project. (Unless there is some `rebar2` plugin that helps in)
What I've seen others doing is they convert their deps in `rebar.config.script` if they detect a `rebar2` env.
Should we add this `rebar2` support, or nobody uses that old tool any more?